### ISSUE TITLE
[steamshovel] RangeSetting compile error (Trac #1059)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1053]
Fails to compile on SL6 cvmfs py2-v1:

```text
[ 80%] Building CXX object steamshovel/CMakeFiles/shovelart.dir/private/shovelart/artists/CoordinateSystem.cpp.o
In file included from
    /scratch/dschultz/offline/src/steamshovel/private/shovelart/Artist.h:25,
from
    /scratch/dschultz/offline/src/steamshovel/private/shovelart/artists/Coordin
    ateSystem.h:4,
from /scratch/dschultz/offline/src/steamshovel/private/shovelart/artists/CoordinateSystem.cpp:8:
    /scratch/dschultz/offline/src/steamshovel/private/shovelart/RangeSetting.h: In
    constructor ‘RangeSetting::RangeSetting(float, float, int, double)’:
/scratch/dschultz/offline/src/steamshovel/private/shovelart/RangeSetting.h:40:
    error: class ‘RangeSetting’ does not have any field named ‘
    ComplexSetting’
/scratch/dschultz/offline/src/steamshovel/private/shovelart/RangeSetting.h:43:
    error: No match for ‘ComplexSetting<float>::ComplexSetting()’
/scratch/dschultz/offline/src/steamshovel/private/shovelart/ComplexSetting.h:16:
    note: candidates are: ComplexSetting<float>::ComplexSetting(float)
/scratch/dschultz/offline/src/steamshovel/private/shovelart/ComplexSetting.h:12:
    note: ComplexSetting<float>::ComplexSetting(const ComplexSetting<float> &)
/scratch/dschultz/offline/src/steamshovel/private/shovelart/RangeSetting.h: In
    constructor ‘RangeSetting::RangeSetting(float, float, int, float)’:
```

Migrated from https://code.icecube.wisc.edu/ticket/1059
```json
{
    "status": "closed", 
    "changetime": "2015-08-10T20:31:17", 
    "description": "Fails to compile on SL6 cvmfs py2-v1:\n\n{{{\n[ 80%] Building CXX object steamshovel/CMakeFiles/shovelart.dir/private/shovelart/artists/CoordinateSystem.cpp.o\nIn file included from\n    /scratch/dschultz/offline/src/steamshovel/private/shovelart/Artist.h:25,\nfrom\n    /scratch/dschultz/offline/src/steamshovel/private/shovelart/artists/Coordin\n    ateSystem.h:4,\nfrom /scratch/dschultz/offline/src/steamshovel/private/shovelart/artists/CoordinateSystem.cpp:8:\n    /scratch/dschultz/offline/src/steamshovel/private/shovelart/RangeSetting.h: In\n    constructor \u2018RangeSetting::RangeSetting(float, float, int, double)\u2019:\n/scratch/dschultz/offline/src/steamshovel/private/shovelart/RangeSetting.h:40:\n    error: class \u2018RangeSetting\u2019 does not have any field named \u2018\n    ComplexSetting\u2019\n/scratch/dschultz/offline/src/steamshovel/private/shovelart/RangeSetting.h:43:\n    error: No match for \u2018ComplexSetting<float>::ComplexSetting()\u2019\n/scratch/dschultz/offline/src/steamshovel/private/shovelart/ComplexSetting.h:16:\n    note: candidates are: ComplexSetting<float>::ComplexSetting(float)\n/scratch/dschultz/offline/src/steamshovel/private/shovelart/ComplexSetting.h:12:\n    note: ComplexSetting<float>::ComplexSetting(const ComplexSetting<float> &)\n/scratch/dschultz/offline/src/steamshovel/private/shovelart/RangeSetting.h: In\n    constructor \u2018RangeSetting::RangeSetting(float, float, int, float)\u2019:\n}}}", 
    "reporter": "david.schultz", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1439238677802127", 
    "component": "combo core", 
    "summary": "[steamshovel] RangeSetting compile error", 
    "priority": "blocker", 
    "keywords": "", 
    "time": "2015-07-20T17:26:52", 
    "milestone": "", 
    "owner": "hdembinski", 
    "type": "defect"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20150720T17:47:33: 
* hdembinski commented:

> In changeset 135342:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="135342"
> fixes #1053
> ```


* hdembinski changed resolution from "" to "fixed"
* hdembinski changed status from "new" to "closed"

**Comment by icecube-trac** [comment]
Trac update at 20150720T18:08:14: hdembinski commented:

> In changeset 135343:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="135343"
> fixes #1053
> ```

**Comment by icecube-trac** [comment]
Trac update at 20150810T20:31:17: 
* olivas changed milestone from "offline V15-07-00" to ""
* olivas commented:

> Milestone offline V15-07-00 deleted
